pkg/apis/cassandra.rook.io/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+// #nosec G601/
```

Comment:
Can we generate without the trailing backslash?

Reply:
yes, Done.